avocado/core/messages.py

```diff
@@ -340,7 +340,8 @@ class StdoutMessageHandler(BaseRunningMessageHandler):
     """
     Handler for stdout message.
 
-    It will save the stdout to the stdout and debug file in the task directory.
+    It will save the stdout to the debug file in the task directory,
+    and optionally (depending on "log_only" value) to the stdout file.
```

Comment:
Can you please add `log_only` variable in to the docstring parameters and update the docstring example.  This docstring is used in our documentation to describe what each message can do.

Reply:
Oh, you meant to the docstring parameters, I did it for the example only.  Hang on.

Reply:
Done for both stdout and stderr in e8f20cbf6f271ad3c072a2b182330aaf14d45695.